src/views/virtualmachines/list/components/VirtualMachineListSummary/VirtualMachineListSummary.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FC, useState } from 'react';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { ALL_PROJECTS } from '@kubevirt-utils/hooks/constants';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { VM_STATUS } from '@kubevirt-utils/resources/vm/utils/vmStatus';
+import { FilterValue } from '@openshift-console/dynamic-plugin-sdk';
+import { ERROR } from '@overview/OverviewTab/vm-statuses-card/utils/constants';
+import { getVMStatuses } from '@overview/OverviewTab/vm-statuses-card/utils/utils';
+import VMStatusItem from '@overview/OverviewTab/vm-statuses-card/VMStatusItem';
+import {
+  Card,
+  CardHeader,
+  CardTitle,
+  Divider,
+  ExpandableSection,
+  Grid,
+  Text,
+} from '@patternfly/react-core';
+import { ProjectDiagramIcon } from '@patternfly/react-icons';
+
+import './VirtualMachineListSummary.scss';
+
+type VirtualMachineListSummaryProps = {
+  namespace: string;
+  onFilterChange?: (type: string, value: FilterValue) => void;
+  vms: V1VirtualMachine[];
+};
+
+const VirtualMachineListSummary: FC<VirtualMachineListSummaryProps> = ({
+  namespace,
+  onFilterChange,
+  vms,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const [isExpanded, setIsExpanded] = useState<boolean>(true);
+
+  const { primaryStatuses } = getVMStatuses(vms || []);
+
+  return (
+    <ExpandableSection
+      toggleContent={
+        <Text className="vm-list-summary__expand-section-toggle" component="h3">
+          <ProjectDiagramIcon className="vm-list-summary__expand-section-toggle-icon" />{' '}
+          {namespace ? namespace : ALL_PROJECTS}
```

Comment:
nitpick: I think this could be replaced with the nullish coalescing operator.

`namespace ?? ALL_PROJECTS`